### PR TITLE
Addressing several warnings

### PR DESCRIPTION
**Pull request by arjantijms**
Signed-off-by: arjantijms <arjan.tijms@gmail.com>

### CODE REVIEW

**Comment by arjantijms** [comment]
jenkins test please

File: nucleus/security/core/src/main/java/com/sun/enterprise/security/common/ClientSecurityContext.java [hunk 1]
```diff
@@ -40,129 +40,67 @@
...
-    public static final String IIOP_CLIENT_PER_THREAD_FLAG =
-        "com.sun.appserv.iiopclient.perthreadauth";
+    private static final long serialVersionUID = -8079501498521266505L;
+
+    public static final String IIOP_CLIENT_PER_THREAD_FLAG = "com.sun.appserv.iiopclient.perthreadauth";
 
     // Bug Id: 4787940
-    private static final boolean isPerThreadAuth = 
-            Boolean.getBoolean(IIOP_CLIENT_PER_THREAD_FLAG);
+    private static final boolean isPerThreadAuth = Boolean.getBoolean(IIOP_CLIENT_PER_THREAD_FLAG);
```
Comment (MarkWareham) [inline nucleus/security/core/src/main/java/com/sun/enterprise/security/common/ClientSecurityContext.java]: Capitalise constant

File: nucleus/security/core/src/main/java/com/sun/enterprise/security/common/ClientSecurityContext.java [hunk 2]
```diff
@@ -40,129 +40,67 @@
```
Comment (MarkWareham) [inline nucleus/security/core/src/main/java/com/sun/enterprise/security/common/ClientSecurityContext.java]: Requires copyright

File: nucleus/security/core/src/main/java/com/sun/enterprise/security/auth/login/ClientPasswordLoginModule.java [hunk 3]
```diff
@@ -133,155 +136,160 @@ public void initialize(Subject subject, CallbackHandler callbackHandler, Map sha
...
-        if (pswd == null) {
+        if (incomingPassword == null) {
             if (System.getProperty(LOGIN_PASSWORD) != null) {
-                pswd = System.getProperty(LOGIN_PASSWORD).toCharArray();
+                incomingPassword = System.getProperty(LOGIN_PASSWORD).toCharArray();
             }
             doSet = true;
         }
 
         if (doSet) {
```
Comment (MarkWareham) [inline nucleus/security/core/src/main/java/com/sun/enterprise/security/auth/login/ClientPasswordLoginModule.java]: may be a little easier to understand as if(incomingUsername == null || incomingPassword ==null) { and do away with "doSet" and assignments.

File: nucleus/security/core/src/main/java/com/sun/enterprise/security/auth/login/ClientPasswordLoginModule.java [hunk 4]
```diff
@@ -133,155 +136,160 @@ public void initialize(Subject subject, CallbackHandler callbackHandler, Map sha
...
-            password = null;
-            commitSucceeded = true;
-            return true;
+        _logger.log(FINE, "\t\t[ClientPasswordLoginModule] " + "added PrincipalImpl to Subject");
+
+        String realm = DEFAULT_REALMNAME;
+
+        // 2. Add a PasswordCredential (containing the same username as the Principal) to the Subject
+
+        PasswordCredential passwordCredential = new PasswordCredential(username, password, realm);
```
Comment (MarkWareham) [inline nucleus/security/core/src/main/java/com/sun/enterprise/security/auth/login/ClientPasswordLoginModule.java]: realm is only used once. Could do away with extra variable.

**Pull request merged**